HelpSource/Guides/UsingQuarks.schelp

```diff
@@ -5,7 +5,28 @@ related:: Classes/Quarks, Classes/Quark, Classes/Git
 
 Quarks are packages of SuperCollider code containing classes, extension methods, documentation and server UGen plugins. The Quarks class manages downloading these packages and installing or uninstalling them.
 
-section:: Installing quarks
+warning::
+Because individual users or groups develop their own Quark for different purposes, installing multiple packages can cause problems because they conflict with each other. We recommend that you select and install only the packages that you need. It is also recommended that you install one Quark package at a time to make it easier to troubleshoot problems as they arise.
+::
+
+note::
+If you have trouble finding the Quarks package you need using the Quarks installation methods in this document, you may find the Quarks package information in the following encyclopaedia useful:
+link::https://baryon.supercollider.online::
+::
+
+section::Preliminary stages
+
+To use Quark, you shoud install git software from link::https://git-scm.com::.
+list::
+##Windows users can download and install the appropriate installation file for their environment from the above web page.
+##Linux users can copy the appropriate terminal commands for their environment and run them in a terminal window.
+##For Mac users, we recommend installing git from Homebrew, which can be done in one of the many ways described on the web page above. If you are unsure whether you have Homebrew installed, follow the instructions for installing Homebrew first. You may also need to install 'Xcode Command Line Tools' if a window pops up asking you to install it.
```

Comment:
from Homebrew, as described at link::https://git-scm.com/download/mac::. If you are unsure...

Reply:
thanks, done!